src/api/api.cpp

```diff
@@ -135,6 +123,26 @@ static rgl_status_t rglSafeCall(Fn fn)
 	return updateAPIState(RGL_SUCCESS);
 }
 
+static void rglLazyInit()
+{
+	static bool initCalled = false;
+	if (initCalled) {
+		return;
+	}
+	initCalled = true;
+	rgl_status_t initStatus = rglSafeCall([&]() {
+		Logger::getOrCreate();
+		Optix::getOrCreate();
+		if (isCompiledWithAutoTape()) {
+			tapeRecord.emplace(std::filesystem::path(RGL_AUTO_TAPE_PATH));
```

Comment:
I'd leave log info that auto tape has started on {path}

Reply:
Good idea!